.github/workflows/release-uv.yml

```diff
@@ -0,0 +1,75 @@
+name: Release new UV-based container images
+
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  BASE_IMAGE_NAME: plone/server
```

Comment:
I forget, what is the plone/server image (instead of plone/plone-backend)? I see it on GHCR at https://github.com/plone/plone-backend/pkgs/container/server but not on dockerhub

Reply:
@ericof Maybe we should call them plone/python-dev and plone/python-prod? They don't have any server in them, and Volto also has a server...